e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-to-an-existing-payment.spec.js

```diff
@@ -149,4 +163,93 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can add fee records to existing paymen
       pages.utilisationReportAddToAnExistingPaymentPage.errorSummary().contains('Select a payment to add the fee or fees to');
     });
   });
+
+  it('should add the selected fees to the payments and update the fee record status after clicking the continue button', () => {
+    cy.task(NODE_TASKS.REMOVE_ALL_UTILISATION_REPORTS_FROM_DB);
+    cy.task(NODE_TASKS.REMOVE_ALL_PAYMENTS_FROM_DB);
+    cy.task(NODE_TASKS.INSERT_UTILISATION_REPORTS_INTO_DB, [report]);
+    cy.task(NODE_TASKS.INSERT_FEE_RECORDS_INTO_DB, [firstFeeRecord, secondFeeRecord]);
+
+    navigateToAddToExistingPaymentScreenForFirstFeeRecord();
+
+    pages.utilisationReportAddToAnExistingPaymentPage.continueButton().click();
+
+    pages.utilisationReportPage.premiumPaymentsTab.getPaymentLink(PAYMENT_ID_ONE).should('contain', 'GBP 450.00');
+    pages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable
+      .getStatus(FEE_RECORD_ID_ONE)
+      .invoke('text')
+      .then((text) => {
+        expect(text.trim()).to.equal('MATCH');
+      });
+  });
+
+  it('should redirect the user to the premium payments page when user clicks the back or cancel links and persist the selected fees', () => {
+    const FEE_RECORD_ID_THREE = '33';
+    const FEE_RECORD_ID_FOUR = '44';
+    const thirdFeeRecord = FeeRecordEntityMockBuilder.forReport(report)
+      .withId(FEE_RECORD_ID_THREE)
+      .withFacilityId('33333333')
+      .withExporter('Exporter 3')
+      .withPaymentCurrency(PAYMENT_CURRENCY)
+      .withFeesPaidToUkefForThePeriod(300)
+      .withFeesPaidToUkefForThePeriodCurrency('EUR')
+      .withPaymentExchangeRate(0.5)
+      .withStatus('TO_DO')
```

Comment:
Do we need a jira ticket to address this TODO?

Reply:
Replaced all literal status references in the arrange blocks with their enum/const value (`FEE_RECORD_STATUS.TO_DO` for this case)